src/Graph.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * Transform array to graph
+ */
+
+/**
+ * TYPES
+ */
+
+type node = {
+  x: Number;
+  y: Number;
+  value: Number;
+  isExit: Boolean;
+};
+
+type edge = {
+  from: string;
+  to: string;
+  direction: string;
+};
+
+type Graph = {
+  nodes: {
+    [coordinates: string]: node;
+  };
+  edges: Array<edge>;
+};
+
+/**
+ * FUNCTIONS
+ */
+
+export function boardToGraph(board: Array<Array<Number>>): Graph | undefined {
+
+  if (board.length < 1) return undefined;
+
+  let graphFromBoard: Graph = newBlankGraph();
+
+
+  const verticalLines: Number = board.length;
+    const horizontalLines: Number = board[0].length;
+  
+
+    for (let hIndex = 0; hIndex < horizontalLines; hIndex++){
+        for (let vIndex = 0; vIndex < verticalLines; vIndex++){
+
+            graphFromBoard = addBoardBoundaries(graphFromBoard, board, hIndex, vIndex)
```

Comment:
the 'graphFromBoard' name here is a bit misleading. Why don't you just name it 'graph'?

Reply:
I admit it is not as well named as i thought. It has been renamed in the [82db828 commit](https://github.com/marmelab/kuba-integration/commit/82db8286f61a76a34787eaa68ea9e0598db448e8)